src/Symfony/Component/Validator/Tests/Constraints/FileTest.php

```diff
@@ -101,8 +101,8 @@ public static function provideValidSizes()
             ['3M', 3000000, false],
             ['1gi', 1073741824, true],
             ['1GI', 1073741824, true],
-            ['4g', 4000000000, false],
-            ['4G', 4000000000, false],
+            ['2g', 2000000000, false],
+            ['2G', 2000000000, false],
```

Comment:
Should we make this change on 5.4 to avoid conflicts when merging up?

Reply:
I agree, I'm opening the PR 👍